src/frontend/sharedComponents/Map/MapView.js

```diff
@@ -391,7 +393,7 @@ class MapView extends React.Component<Props, State> {
             )}
             {styleType === "fallback" ? <OfflineMapLayers /> : null}
             {locationServicesEnabled ? (
-              <MapboxGL.UserLocation
+              <UserLocation
                 visible={isFocused}
                 minDisplacement={MIN_DISPLACEMENT}
```

Comment:
I  converted this to a functional component so I could use the `useSettingsValue` hook